lua/wikis/pubg/Infobox/Person/Player/Custom.lua

```diff
@@ -54,7 +55,7 @@ end
 ---@return table
 function CustomPlayer:adjustLPDB(lpdbData, args)
 	if String.isNotEmpty(args.team2) then
-		lpdbData.extradata.team2 = mw.ext.TeamTemplate.raw(args.team2).page
+		lpdbData.extradata.team2 = TeamTemplate.getRaw(args.team2).page
```

Comment:
why not use `TeamTemplate.getPageName`?
(same in several of the other cases too)